package.json

```diff
@@ -70,99 +70,100 @@
     "deploy-dev": "gh-pages -d styleguide -e dev -m \"[skip ci] Updates\""
   },
   "devDependencies": {
-    "@babel/core": "7.17.7",
-    "@babel/plugin-transform-runtime": "7.17.0",
-    "@babel/preset-env": "7.16.11",
-    "@babel/preset-react": "7.16.7",
+    "@babel/core": "7.18.9",
+    "@babel/plugin-transform-runtime": "7.18.9",
+    "@babel/preset-env": "7.18.9",
+    "@babel/preset-react": "7.18.6",
     "@reach/rect": "0.16.0",
     "@rollup/plugin-babel": "5.3.1",
-    "@rollup/plugin-commonjs": "21.0.2",
-    "@rollup/plugin-node-resolve": "13.1.3",
-    "@stylelint/postcss-css-in-js": "0.37.2",
-    "@testing-library/jest-dom": "5.16.2",
+    "@rollup/plugin-commonjs": "22.0.1",
+    "@rollup/plugin-node-resolve": "13.3.0",
+    "@stylelint/postcss-css-in-js": "0.38.0",
+    "@testing-library/jest-dom": "5.16.4",
     "@testing-library/react": "12.1.4",
     "@types/classnames": "2.3.1",
-    "@types/jest": "27.4.1",
-    "@types/jest-axe": "3.5.3",
+    "@types/jest": "28.1.6",
```

Comment:
There are some major release updates in these dependencies, how has this been tested?

Reply:
Styleguidist Firefox, Chrome, Safari. CRA Firefox, Chrome, Safari.